gulpfile.js

```diff
@@ -109,6 +109,16 @@ gulp.task('copy-fonts', () => {
   return gulp.src('src/fonts/**/*').pipe(gulp.dest('dist/fonts/'));
 });
 
+// Copies content default locale into it's own folder on prod.
```

Comment:
Let's mention here we need it because of FB hosting lang redirects.

Reply:
+1